src/GitVersion.Core.Tests/IntegrationTests/JustSomeTestScenarios.cs

```diff
@@ -0,0 +1,228 @@
+using GitTools.Testing;
+using GitVersion.Core.Tests.Helpers;
+using GitVersion.Model.Configuration;
+using GitVersion.VersionCalculation;
+using NUnit.Framework;
+
+namespace GitVersion.Core.Tests.IntegrationTests;
+
+[TestFixture]
+public class JustSomeTestScenarios : TestBase
```

Comment:
We need a better name than `JustSomeTestScenarios`. :)

Reply:
I fully agree with you. I need to rename it and give the scenarios a good name. It's pending